### ISSUE TITLE
Release new flatpak version with the epic store fix

### ISSUE THREAD

**Issue by minimeee** [flathub/net.lutris.Lutris#297]
Epic authentication in Lutris broke 1 month ago and has been fixed in the meantime: https://github.com/lutris/lutris/issues/4472.

However, it's still broken in the flatpak version, as it hasn't been updated in the last two months. Please release a new flatpak beta with the EGS fix, so we can finally use epic games integration on the SteamDeck again. 

Btw, I tried to run the git version on my SteamDeck (as I do on my desktop) but it's not possible due to missing `webkit2gtk` (which can't be installed).

**Comment by danieljohnson2** [comment]
Your distro might have a 'lutris-git' package, which is basically that unstable Lutris. But I don't know if that's a great thing to use- it's going to be broken sometimes.

However, I contacted @strycore  and he's amenable to a Lutris release "soon". So hopefully you won't have to wait too long for an update with these fixes.

**Comment by minimeee** [comment]
> Your distro might have a 'lutris-git' package

Thx, but on the steam deck, you only have flatpaks available. Well, technically you can install arch packages if you disable the OS write protection, but they would be removed during the next update. Maybe it's a workaround for other distros, though. On my desktop, I'm using the git version of Lutris and it works very well. Would be great to have that option as a flatpak too.

Out of curiosity: why is it so hard to update the flatpak version? At first glance it looks like it would be enough to change this commit to sth. more recent:
https://github.com/flathub/net.lutris.Lutris/blob/1d560df42f3b73216855e0d822b6981b1167eb96/net.lutris.Lutris.yml#L626

But maybe that would cause dependency problems?

**Comment by NightHammer1000** [comment]
@minimeee 
Better off just using the Heroic Flatpak. 
The bugfix time on the Lutris Flatpak is horrible. There is still an Issue with wine-lutris that does not let you run games of SD Card that is still being ignored. They don't seem very interested in supporting the Deck. Sadly.
Heroic had the fix up and running in less than an day.

**Comment by minimeee** [comment]
Thanks @NightHammer1000.  I already tried Heroic on the Deck and on my PC. Always went back to Lutris because of various issues (e. g. games not starting or running very slow). I also prefer the configurability and high number of services supported by Lutris. You're right about the slow bugfix time with flatpak and the EGS issue, though.

> There is still an Issue with wine-lutris

I only use the [GE versions of WINE](https://github.com/GloriousEggroll/wine-ge-custom) with Lutris.

**Comment by NightHammer1000** [comment]
Yeah. But lutris ignores the prefered Wine settings when installing stuff and defaults back to the broken wine-lutris

**Comment by matheusfaustino** [comment]
sorry to bother you guys, but any update related to this? So, currently, the only solution for the epic sync is to either change the file manually or not using flatpak, right?

**Comment by minimeee** [comment]
@matheusfaustino: you can also build the Flatpak yourself by following the instructions [here](https://github.com/flathub/net.lutris.Lutris#building) and the workarounds [here](https://github.com/flathub/net.lutris.Lutris/issues/304).

In order to get a recent Lutris version, you only have to replace the commit hash in this line: https://github.com/flathub/net.lutris.Lutris/blob/1d560df42f3b73216855e0d822b6981b1167eb96/net.lutris.Lutris.yml#L626

I built one successfully based on commit `09feb408a5ecc69d5820fed61930e84bbd893c72` (which corresponds to release [v0.5.12-beta1](https://github.com/lutris/lutris/releases/tag/v0.5.12-beta1)). But note that you may end up with incompatible dependencies for other commits.

Lutris is great imo, but it's really a shame that they have such a slow (and unpredictable) release cycle. The desktop version is also outdated most of the time, that's why I've switched to the `git` version on my PC long time ago...

**Comment by matheusfaustino** [comment]
@minimeee thank you, I will try. I did a workaround which it was manually copying the file from the new version to the flatpak directory, and that way I was able to fetch the list. But, your way is more complete. Thank you for explaning and for the links ^^ 
Have a nice day
